src/main/java/CurrencyExchangeFrame.java

```diff
@@ -31,9 +35,12 @@ public class CurrencyExchangeFrame extends JFrame implements ItemListener {
     private String[] symbolsArray;
     private String[] descriptionsArray;
 
-    public CurrencyExchangeFrame() {
+    @Inject
+    public CurrencyExchangeFrame(CurrencyExchangePresenter presenter) {
```

Comment:
Put this and the Presenter in a package.

We shouldn't be using the default package.